### DISCUSSION TITLE
Modal scroll bar long content

### DISCUSSION THREAD

**Discussion by dungdc40** [saadeghi/daisyui#642] [Q&A] [answered]
When there is long content on the modal, it display a scrollbar

I want it to display scrollbar on the browser instead of the modal, similar to behavior of bootstrap
https://getbootstrap.com/docs/4.0/components/modal/
(Scrolling long content section)

Can anyone help?

**Comment by saadeghi** [comment, marked as answer]
Example: https://play.tailwindcss.com/kjilvmOpp8

**Comment by dungdc40** [comment]
thanks man

**Comment by dungdc40** [comment]
@saadeghi Can you help me a bit more :)
If the page already long, then the modal add additional scrollbar, can it take over the page scrollbar (as bootstrap example)

Here is double scrollbar example: https://play.tailwindcss.com/PI9euVRMdW

**Comment by dungdc40** [comment]
I figured it out, I have to set body's overflow: hidden when my modal is hidden, thanks :)
